### ISSUE TITLE
Docker and ts-node

### ISSUE THREAD

**Issue by crivera** [TypeStrong/ts-node#714]
Hi

I have a mini project where the package.json looks like this: 
```
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "test.js",
  "scripts": {
    "meow": "ts-node -e 'console.log(\"test\")'"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "ts-node": "^7.0.1",
    "typescript": "^3.1.3"
  }
}
```

When I do `npm install` and then run `npm run meow` i see the print out of test.

Now when I do put this now in a docker image with the docker file being:

```
FROM node:latest

COPY . . 

RUN npm install

ENTRYPOINT ["npm"]

CMD ["run", "meow"]
```

When i do `docker build -t test .` and then run `docker run -it test` the process just seems stuck. Nothing happens.

Have you guys experienced that before? Do I need a special node version?

Thanks
Chris

**Comment by crivera** [comment]
When I update the meow command to: node -e 'console.log(\"test\")' and build a new docker image it works fine.

**Comment by blakeembrey** [comment]
Do you have a `tsconfig.json` file?

**Comment by crivera** [comment]
No...

**Comment by blakeembrey** [comment]
You don’t need one, but probably should have one. I’m taking a guess that it may be related to https://github.com/TypeStrong/ts-node/issues/690.

**Comment by blakeembrey** [comment]
Closing this for now, should be resolved with #690 in 8.0.

**Comment by crivera** [comment]
It seems like you need a WORKDIR amd then it will execute.
